release_notes/48-release-notes.adoc

```diff
@@ -42,7 +42,33 @@ Policy::
 
 This release adds improvements related to the following components and concepts:
 
+//ROX-20769
+[id="central-db-postgresql_15_{context}"]
+=== Central DB uses PostgreSQL 15
 
+Central DB component is using PostgreSQL 15 now, and {product-title-short} 4.8
+supports this version for external databases as well. A new installation with
+an internal database will use this version by default now. When upgrading an
+existing cluster to {product-title-short} 4.8, Central DB will do an in-place
+upgrade of the data collected so far.
+
+[NOTE]
+====
+To make the database upgrade process smooth, we recommend:
+
+* Take a backup before upgrading to {product-title-short} 4.8
+
+* If not using the Operator: check disk space available for the database
+`rox_central_postgres_available_size_bytes`. For the purposes of the upgrade,
+it should be double of already consumed disk space
+`rox_central_postgres_total_size_bytes`. If that is not the case, extend the
+database PVC.
+
+* Do not interrupt the upgrade procedure. If that happens, a manual
+intervention could be required to proceed.
+
+Depending on the amount of data the upgrade may take extra time to finish.
```

Comment:
```suggestion
Depending on the amount of data, the upgrade can take extra time to finish.
```
* Move this sentence to go with previous paragraph that I am recommending to change into an important note.
* If you are curious, you can see the guidance on "may" in [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#may) (use redhat ID to log in)